helpers/match_manipulator.py

```diff
@@ -119,7 +119,8 @@ def updateMerge(self, new_match, old_match, auto_union=True):
 
         auto_union_attrs = [
             "tba_videos",
-            "youtube_videos"
+            "youtube_videos",
+            "vimeo_videos"
```

Comment:
I agree, especially if we want to ship #1389 and link Medias to other models in the future.